docs/wmco-prerequisites.md

```diff
@@ -50,3 +50,18 @@ consistent across all supported cloud providers.
 
 ## vSphere Specific Requirements
 Please refer to [VMware vSphere pre-requisites](vsphere-prerequisites.md).
+
+## Supported Platforms for Community Operator
+| Cloud Provider            | Available for Trial |
```

Comment:
- `Supported Platforms` conflicts with `Available for Trial`. Moreover, do you really need this column if it is defaulting to yes.
- Shouldn't this be in the community CSV?

Reply:
> Supported Platforms conflicts with Available for Trial. Moreover, do you really need this column if it is defaulting to yes.
I'll remove the Yes column and change table title to Community Operator Platforms Available for Trial
>Shouldn't this be in the community CSV?
Where does that live?

Reply:
> > Supported Platforms conflicts with Available for Trial. Moreover, do you really need this column if it is defaulting to yes.
> 
> I'll remove the Yes column and change table title to Community Operator Platforms Available for Trial

I am not sure if "Trial" is correct in this context. Trial implies that after trying it out there is some way to "buy" the product. Please remind me what was the decision we made for community regarding Supported vs Tech Preview. That is what needs to be called out for the community operator.

> > Shouldn't this be in the community CSV?
> 
> Where does that live?

Please work with @sebsoto on that.

Reply:
>Please remind me what was the decision we made for community regarding Supported vs Tech Preview.

We decided that we wouldn't explicitly declare support for these platforms for the release, but users are welcome to try it out with our community operator. Anand asked me to add this table to the community docs per this comment https://docs.google.com/document/d/1x74Bf7e-9DKxX_wltR9F340rhwk3xu9qSJ1YSI5ZeHY/edit?disco=AAAAPc8GM-4

>Please work with @sebsoto on that.
Will do!

Reply:
Thanks for refreshing my memory. I now see where the trial verbiage comes from. I would rather say "works on". Everything on the community operator is preview by default. Don't want to give users the impression that they go from trial to supported on the community version.